core/jobs/batch_jobs/rejecting_suggestion_for_invalid_content_ids_jobs.py

```diff
@@ -0,0 +1,586 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rejecting suggestions whose content_id no longer exists and
+updating the translation content.
+"""
+
+from __future__ import annotations
+
+from core import feconf
+from core import utils
+from core.domain import exp_fetchers
+from core.jobs import base_jobs
+from core.jobs.io import ndb_io
+from core.jobs.transforms import job_result_transforms
+from core.jobs.types import job_run_result
+from core.platform import models
+
+import apache_beam as beam
+import bs4
+import json
+
+from typing import List, Union
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import exp_models
+    from mypy_imports import suggestion_models
+
+(exp_models, suggestion_models) = models.Registry.import_models([
+    models.Names.EXPLORATION, models.Names.SUGGESTION
+])
+
+datastore_services = models.Registry.import_datastore_services()
+
+
+class RejectSuggestionWithMissingContentIdMigrationJob(base_jobs.JobBase):
+    """Job that rejects the suggestions for missing content ids and
+    updates the RTE content.
+    """
+
+    empty_values = [
+        '&quot;&quot;', '\\"&quot;&quot;\\"', '', '\'\'', '\"\"', '<p></p>']
```

Comment:
Please move this into some constant, I see you redefining this everywhere.

Reply:
Removed this part as the changes were present in the conversion function and took directly from there. Initially I opened other PR for this and these changes were required there.
Thanks